src/backend/model/data/migrations/20220804075124_add_password_column_to_owner.js

```diff
@@ -0,0 +1,21 @@
+const TABLE_NAME = "owners";
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void[]> }
+ */
+exports.up = async function (knex) {
+  await knex.schema.alterTable(TABLE_NAME, function (table) {
```

Comment:
Add return keyword.
Don't need to add await keyword, because the jsdoc says `@returns { Promise<void[]> }`